news/gh-11012-conda-canary

```diff
@@ -0,0 +1,26 @@
+### Enhancements
+
+* Automatically build the conda package upon the successful merge into the
+  master branch and upload it to the conda-canary channel on anaconda.org.
+
+  To try conda out simply run:
+
+  ```
+  conda install -c conda-canary conda
```

Comment:
Option should be `-c conda-canary/label/dev`

Reply:
Oh, huh, good catch!